packages/graphql/src/registry.ts

```diff
@@ -91,9 +103,71 @@ export class GraphQLTypeRegistry {
     return gqlEnum;
   }
 
+  private computeModelFields(tspModel: Model): GraphQLFieldConfigMap<any, any> {
```

Comment:
From what I've seen elsewhere in the TypeSpec codebase (and it makes sense to me), we should use [JavaScript's `#private` properties](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_properties) over TypeScript's (pseudo-)`private` properties.